scripts/performance/common.py

```diff
@@ -87,6 +87,28 @@ def get_artifacts_directory() -> str:
     '''
     return os.path.join(get_repo_root_path(), 'artifacts')
 
+def rename_upload_files(files, helix_workitem_id) -> None:
```

Comment:
double super nit: maybe:
```suggestion
def rename_upload_files(files, differentiator) -> None:
```